tooling/monitoring-dashboard/README.md

```diff
@@ -0,0 +1,34 @@
+# Monitoring Dashboard for WSO2 Micro Integrator
+
+## Building from the source
+
+- ### Setting up the development environment
+1. Install Node.js [8.X.X](https://nodejs.org/en/download/releases/)
+2. Fork the [repository](https://github.com/wso2/micro-integrator)
+3. Clone your fork into any directory
+4. `cd` into cloned directory and then cd into `micro-integrator/tooling/monitoring-dashboard`.
+5. Perform a maven build ```mvn clean install```
+6. wso2mi-monitoring-dashboard-version.zip can be found in
+ `micro-integrator/tooling/monitoring-dashboard/distribution/target`
+ 
+- ### Running
+- Extract the compressed archive generated to a desired location.
+    ```
+    cd to the DASHBOARD-HOME/bin
+    
+    Execute `dashboard.sh or dashboard.bat as appropriate.
+    ```
+
+
+- Load the login page with the dashboard context. i.e: https://localhost:9743/dashboad
+- Please note that in order to connect the monitoring dashboard to a micro integrator instance, the
+  said instance should have the management api enabled. This can be done by starting the MI with 
+  the -DenableManagementApi property.
+  
+- ### How to Enable the Management API
+    By default the Management Api is disabled. To use the Management Api you must use the system property `-DenableManagementApi` when starting the micro integrator
```

Comment:
Please note that Management Api is disabled by default. It could be enabled by issueing the following command from <DASHBOARD>/bin directory. And I think we need to provie the command as @Nirothipan has mentioned.